ios/FluentUI/Extensions/EnvironmentValues+Extensions.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+import SwiftUI
```

Comment:
Thinking about the name of this file. We have a few other places where we extend EnvironmentValues, and it's usually done in the context of what we're intending to add to the app (e.g. `FluentTheme.swift`).

Do you think we should move those all here to a single common `EnvironmentValues+Extensions.swift` file, or should this particular extension be renamed to something more explicit about its intent (e.g. `EnvironmentValues+SafeAreaInsets` or the like)?

Reply:
Ah yes thanks for the reminder 🤠 I do still like the idea of it living separately! Just wondering about renaming it to be more specific to its contents.